### DISCUSSION TITLE
Setting on/off - Load the open/activated chat not the rest

### DISCUSSION THREAD

**Discussion by ijjop** [Chatterino/chatterino2#5723] [Ideas]
This would minimize performance use
Text should load easy though many situations I've noticed slowness with Chatterino. Some chats load first and wait needed for other - if possible prioritize the open/activated chat.

Could achieve this by 'Rightclick' and 'Popup Tab' then closing other window with 60+ tabs, though adjusting new window size every time not optimal.

**Comment by pajlada** [comment]
When Chatterino connects to Twitch, we join channels in alphabetical order. We have plans to change this so it prioritizes channels that are in your currently selected tab. Would that solve your use case?
